ui/page/send/layout.go

```diff
@@ -276,11 +278,17 @@ func (pg *Page) toSection(gtx layout.Context) layout.Dimensions {
 					layout.Rigid(func(gtx C) D {
 						return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
 							layout.Rigid(func(gtx C) D {
-								label := pg.Theme.Body2("Exchange rate not fetched")
+								label := pg.Theme.Body2(pg.exchangeError)
```

Comment:
This should just be `"Exchange rate not fetched. Check internet connection."` I explain why below.